src/main/content/_assets/js/guide-static.js

```diff
@@ -16,83 +16,107 @@ $(document).ready(function () {
     var target_width;
     var target_height;
 
-    $("#preamble").detach().insertAfter("#duration_container");
+    $('#preamble').detach().insertAfter('#duration_container');
+
+    function add_prereq(prereq) {
+        var prereq_html = $('<div class=\'prereq_div\'>');
+        var prereq_link = $(
+            '<a class=\'prereq notranslate\' target=\'_blank\'>' +
+                prereq.name +
```

Comment:
XXS concern.  Who provides the data for variable `prereq.name`? If this is user input, we need to sanitize the input since this is building HTML.